airbyte-webapp/src/packages/cloud/lib/domain/cloudWorkspaces/WorkspaceService.ts

```diff
@@ -1,17 +1,6 @@
-import { AirbyteRequestService } from "core/request/AirbyteRequestService";
+import { createWorkspace } from "../../../../../core/request/GeneratedApi";
 
-import { CloudWorkspace } from "./types";
-
-class WorkspaceService extends AirbyteRequestService {
-  get url() {
-    return `workspaces`;
-  }
-
-  public async create(workspaceCreatPayload: { name: string }): Promise<CloudWorkspace> {
-    const workspace = await this.fetch<CloudWorkspace>(`${this.url}/create`, workspaceCreatPayload);
-
-    return workspace;
-  }
+// TODO: Verify this is used
+export class WorkspaceService {
+  public create = createWorkspace;
```

Comment:
This class seems unused. Maybe Cloud's doing some magic I don't know about?

Reply:
This looks dead to me too. The method we use on cloud to create a new workspace is https://github.com/airbytehq/airbyte/blob/master/airbyte-webapp/src/packages/cloud/lib/domain/cloudWorkspaces/CloudWorkspacesService.ts#L52 so I suspect this can be removed.